src/components/widgets/PresentationItem.js

```diff
@@ -80,19 +75,20 @@ const PresentationItem = ({ label, helpText, required, classes, children, width
 
     if (Array.isArray(children) && children.length > 0) {
       value = children.map((item, i) => (<dd className={classes.content} key={i}>{item}</dd>));
-    } else if (!Array.isArray(children) && children) {
+    } else if (!Array.isArray(children) && typeof children !== 'undefined') {
       value = <dd className={classes.content}>{children}</dd>;
     }
 
     return value;
   };
 
+  const medium = md || 8;
```

Comment:
If it is a prop type, would you mind to update PropTypes description and provide a parameter explanation in the parameters list comment?